siliconcompiler/schema/schema_cfg.py

```diff
@@ -18,8 +18,8 @@ def scparam(cfg,
             require=None,
             defvalue=None,
             scope='job',
```

Comment:
The switch to actual types is a great change!! The initial decision to make everything a string was made pre Python API when I had approximately zero years of Python experience so I wanted to keep things uniform:-)

There is at least one schema change (aspect ratio) with defvalue as string that should also be fixed.